packages/components/codemods/__tests__/parseJsx.ts

```diff
@@ -0,0 +1,11 @@
+import ts from "typescript"
+
+// This function is used to parse a stringified JSX element into an AST
+export const parseJsx = (jsx: string): ts.SourceFile =>
```

Comment:
Not a test?

Reply:
It's a util specifically just for tests :) If it's clearer, maybe `__tests__/utils`?

Reply:
Will move it to a utils folder for more clarity :)